ci/ios.groovy

```diff
@@ -1,7 +1,9 @@
 utils = load('ci/utils.groovy')
 
 def plutil(name, value) {
-  utils.nix_sh "plutil -replace ${name} -string ${value} ios/StatusIm/Info.plist"
+  return """
+  plutil -replace ${name} -string ${value} ios/StatusIm/Info.plist
```

Comment:
I think you need a semilcolon for this to work.

Reply:
Indeed, thanks for spotting it!